src/plugins/kibana/public/settings/sections/indices/add_data_steps/pipeline_setup/master.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
It's not used anywhere, and when we are done developing this it should be deleted. It's there for convenience to be copy/pasted into sample_pipeline.json (which is itself there for convenience during development)